README.adoc

```diff
@@ -165,34 +201,62 @@ Or you can run the container with `docker-compose up -d swagger`.
 
 The Swagger UI is available at http://localhost:8080[localhost:8080].
 
-=== Method 2
+==== By browser
 
 There is also a https://chrome.google.com/webstore/detail/swagger-ui-console/ljlmonadebogfjabhkppkoohjkjclfai?hl=en[Chrome extension] you can use instead of running a Docker container.
 
 Paste https://raw.githubusercontent.com/aerogear/mobile-security-service/master/api/swagger.yaml[https://raw.githubusercontent.com/aerogear/mobile-security-service/master/api/swagger.yaml] and press **Explore**.
 
+TIP: Also, you can use the https://petstore.swagger.io/[Demo] tool online for swaggerUI. 
 
-== Building & Testing
+=== Building & Testing
 
 The `Makefile` provides commands for building and testing the code. Some dependencies are required to run these commands.
 
-== Dependencies
+==== Installing the required dependencies
 
 Dependencies may be required to run some of the `Make` commands. Below are instructions on how to install them.
 
-== Creating mock for Interfaces
-This project is using teh dependency https://github.com/matryer/moq[moq]. Follow the steps below to use it.
-
-=== Install the required dependencies
-
 Run the following command.
 
 [source,shell]
 ----
 $ go get -u github.com/matryer/moq
 ----
 
-=== Creating mocks
+NOTE : See all commands available in link:#using-make-commands[Using make commands]
+
+=== Installing and configuring client tool for the database
+
+You can use the https://www.pgadmin.org/[pgadmin] which is the client tool for PostgreSQL to work with the database.
+
+* Download and install the client tool
+
+NOTE: The link to download for MacOS is : https://www.pgadmin.org/download/pgadmin-4-macos/
+
+* Configure the client tool
+
+Following the steps to do this setup.
+
+. Access the tool via the browser. The default link will be `http://127.0.0.1:52263/browser/#`
+. Create a new server connection with the database. Following the image to show how to do it.
+
+
+image::https://user-images.githubusercontent.com/7708031/53171792-9ecf3380-35db-11e9-8de7-4a7df979b38b.png[Create Server,align="center"]
+
+. Add the data required to connect on the database. See the values defined in link:./pkg/config/config.go[config.go]. Following an image with this setup.
```

Comment:
`connect to the database` rather that `connect on the database`

Reply:
Solved 👍 . Really tks.